packages/vite/types/anymatch.d.ts

```diff
@@ -0,0 +1,5 @@
+export type AnymatchFn = (testString: string) => boolean
+export type AnymatchPattern = string | RegExp | AnymatchFn
+type AnymatchMatcher = AnymatchPattern | AnymatchPattern[]
+
+export { AnymatchMatcher as Matcher }
```

Comment:
**question:** Could you explain why the rename is needed?

Reply:
It's copied from https://github.com/micromatch/anymatch/blob/master/index.d.ts.

Reply:
Maybe we don't need rename. I'm just not sure about it.

Reply:
@patak-js Do you have an opinion on this?
Otherwise, count me as approved :)

Reply:
Not bad to keep it the same as upstream